libosmscout-client-qt/src/osmscoutclientqt/PlaneMapRenderer.cpp

```diff
@@ -413,6 +413,7 @@ void PlaneMapRenderer::DrawMap()
 
     if (loadJob->IsFinished()){
```

Comment:
But when job is finished, Close method should be no-op. Or I am miss something here?

Reply:
I see, rendrers are setting `closeOnFinish` to false explicitly, so job is holding read-lock even when it is finished. It makes sense as renderer requires to have corret link to loaded data and used stylesheet...